edge/pkg/edged/config/config.go

```diff
@@ -187,6 +187,8 @@ func ConvertConfigEdgedFlagToConfigKubeletFlag(in *v1alpha2.TailoredKubeletFlag,
 	out.RegisterSchedulable = in.RegisterSchedulable
 	out.KeepTerminatedPodVolumes = in.KeepTerminatedPodVolumes
 	out.SeccompDefault = in.SeccompDefault
+	out.WindowsPriorityClass = in.WindowsPriorityClass
+	//out.WindowsService = in.WindowsService
```

Comment:
Why is WindowsService commented out? If necessary, add a comment to explain.

Reply:
Because if it's 'true', kubelet will installed as service which does not match our needs